### PR TITLE
Binance: fetchPosition, option support

### PR DESCRIPTION
**Pull request by Dan-krm**
Add fetchPosition with option support:
```
binance.fetchPosition (ETH/USDT:USDT-230426-1850-C)
2023-04-26T07:04:05.921Z iteration 0 passed in 715 ms

{
  info: {
    entryPrice: '27.70000000',
    symbol: 'ETH-230426-1850-C',
    side: 'LONG',
    quantity: '0.50000000',
    reducibleQty: '0.50000000',
    markValue: '10.150000000',
    ror: '-0.2671',
    unrealizedPNL: '-3.700000000',
    markPrice: '20.3',
    strikePrice: '1850.00000000',
    positionCost: '13.85000000',
    expiryDate: '1682496000000',
    priceScale: '1',
    quantityScale: '2',
    optionSide: 'CALL',
    quoteAsset: 'USDT',
    time: '1682492646252'
  },
  id: undefined,
  symbol: 'ETH/USDT:USDT-230426-1850-C',
  entryPrice: 27.7,
  markPrice: 20.3,
  notional: 10.15,
  collateral: 13.85,
  unrealizedPnl: -3.7,
  side: 'long',
  contracts: 0.5,
  contractSize: undefined,
  timestamp: 1682492646252,
  datetime: '2023-04-26T07:04:06.252Z',
  hedged: undefined,
  maintenanceMargin: undefined,
  maintenanceMarginPercentage: undefined,
  initialMargin: undefined,
  initialMarginPercentage: undefined,
  leverage: undefined,
  liquidationPrice: undefined,
  marginRatio: undefined,
  marginMode: undefined,
  percentage: undefined
}
```

### CODE REVIEW

File: ts/src/binance.ts [hunk 1]
```diff
@@ -6757,6 +6757,109 @@ export default class binance extends Exchange {
...
+            'initialMargin': undefined,
+            'initialMarginPercentage': undefined,
+            'leverage': undefined,
+            'liquidationPrice': undefined,
+            'marginRatio': undefined,
+            'marginMode': undefined,
+            'percentage': undefined,
+        });
+    }
+
```
Comment (carlosmiei) [inline ts/src/binance.ts:6862]: @Dan-krm symbol is not mandatory, can't we add options support to `fetchPositions` as well? 
image

File: ts/src/binance.ts [hunk 2]
```diff
@@ -6771,8 +6933,10 @@ export default class binance extends Exchange {
             return await this.fetchPositionsRisk (symbols, params);
         } else if (defaultMethod === 'account') {
             return await this.fetchAccountPositions (symbols, params);
+        } else if (defaultMethod === 'option') {
+            return await this.fetchOptionPositions (symbols, params);
```
Comment (carlosmiei) [inline ts/src/binance.ts:6937]: @Dan-krm hum getting a weird error when calling `fetchPositions` without a symbol

```
handleRestResponse:
 binance GET https://eapi.binance.com/eapi/v1/position?timestamp=1682671768982&recvWindow=10000&signature=glV6Jo6ZDSUSokhGwLrq9wzRykrHeNXtsEU%2B6Z5Bjx%2BF83vaIgs3p9ZdLq%2BhL9MDLZzJSWshUscc464hHUTPdmNXeoOenunU0gbg03JwemjHEBBLx%2BIOLv1PKPz6%2BeBrcSL1u%2BUz1NSyKoEyWVRJF%2BQFxeBdoNJW4bC2w%2BtBg8eTVGtwWbR821mQBeSNDt1WZvroFtx1fiZBD8qjYQFY903pESCm7fqRoT4wMkLpQIbvx7zzc4Vri%2BQ0%2BnVtiGdoUET%2FCuKTl4fvxu5QqfZmMreG801aih63idhhAZgVynxgmH0NbMFSk4kSbXrIdzAeQgb3cfO36K3w2nVg0Qx1qQ%3D%3D 400 Bad Request 
ResponseBody:
 {"code":-1130,"msg":"Invalid data sent for a parameter."} 

BadRequest binance {"code":-1130,"msg":"Invalid data sent for a parameter."}
---------------------------------------------------
[BadRequest] binance {"code":-1130,"msg":"Invalid data sent for a parameter."}

    at throwExactlyMatchedException  Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2793  
    at handleErrors                  Users/cjg/Git/ccxt10/ccxt/js/src/binance.js:7728        
    at                               Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:748   
    at processTicksAndRejections     node:internal/process/task_queues:95                    
    at fetch2                        Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2435  
    at request                       Users/cjg/Git/ccxt10/ccxt/js/src/binance.js:7758        
    at fetchOptionPositions          Users/cjg/Git/ccxt10/ccxt/js/src/binance.js:6905        
    at fetchPositions                Users/cjg/Git/ccxt10/ccxt/js/src/binance.js:7010        
    at async run                     Users/cjg/Git/ccxt10/ccxt/examples/js/cli.js:295        

binance {"code":-1130,"msg":"Invalid data sent for a parameter."}
```

Comment (Dan-krm) [reply ts/src/binance.ts:6937]: I'm not able to recreate the error, it's working properly for me. I think errors like this show up on binance options if you don't click on the options tab on the website to enable it before creating an api key, I'm not sure if something like that is what's happening here

```
binance.fetchPositions ()

handleRestResponse:
 binance GET https://eapi.binance.com/eapi/v1/position?timestamp=1682722597072&recvWindow=10000&signature=50d43ec76e7c418957eb3878374c375d183611322edd138d2fc089e30c09cc40 200 OK

ResponseBody:
 [{"entryPrice":"8.40000000","symbol":"ETH-230429-1900-C","side":"LONG","quantity":"0.50000000","reducibleQty":"0.50000000","markValue":"3.400000000","ror":"-0.1904","unrealizedPNL":"-0.800000000","markPrice":"6.8","strikePrice":"1900.00000000","positionCost":"4.20000000","expiryDate":1682755200000,"priceScale":1,"quantityScale":2,"optionSide":"CALL","quoteAsset":"USDT","time":1682722598554}]

2023-04-28T22:56:37.733Z iteration 0 passed in 711 ms

id |                      symbol | entryPrice | markPrice | notional | collateral | unrealizedPnl | side | contracts | contractSize |     timestamp |                 datetime | hedged | maintenanceMargin | maintenanceMarginPercentage | initialMargin | initialMarginPercentage | leverage | liquidationPrice | marginRatio | marginMode | percentage
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   | ETH/USDT:USDT-230429-1900-C |        8.4 |       6.8 |      3.4 |        4.2 |          -0.8 | long |       0.5 |              | 1682722598554 | 2023-04-28T22:56:38.554Z |        |                   |                             |               |                         |          |                  |             |            | 
1 objects
```

Comment (carlosmiei) [reply ts/src/binance.ts:6937]: @Dan-krm np, thanks for confirming it

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**